test/React.js

```diff
@@ -0,0 +1,14 @@
+function getReact() {
+  try {
+    // 0.14.x Import
+    return require('react');
+  } catch (error) {
+    if (error.code !== 'MODULE_NOT_FOUND') {
+      throw error;
+    }
+    // 0.13.x Import
+    return require('react/addons');
+  }
+}
+
+export default getReact();
```

Comment:
This file is isolating the React import from the test so that we can arbitrate between 0.13.x and 0.14.x